internal/backends/winit/event_loop.rs

```diff
@@ -306,14 +298,16 @@ fn process_window_event(
                 _ => None,
             });
             if let Some(text) = key_code.and_then(key_codes::winit_key_to_string) {
```

Comment:
Just fix winit_key_to_string to return a char instead.  (and probaly rename it winit_key_to_char)

Reply:
Good idea!